### PR TITLE
Removes Frankenline Station From Map Vote

### PR DESCRIPTION
**Pull request by west3436**
Frankenline Station was a meme map built around a meme premise and added to the voting list as a regular map. This PR (0% tested) comments out the votability without removing the map. If the map maker would like to reimplement this map as votable on Halloween or something like that, it is possible to do so in the future.

### CODE REVIEW

**Comment by west3436** [comment]
okay maybe 0% testing was a mistake

**Comment by PrimeDSS13** [comment]
I agree, I've noticed that the map kills pop whenever it rolls. It's a novel idea, but it is an absolute meme. Fits as a bus map, though.

File: code/datums/next_map.dm [hunk 1]
```diff
@@ -80,10 +80,10 @@
 	path = "Island"
 	min_players = 25
 	
-/datum/next_map/line
-	name = "Frankenline Station"
-	path = "line"
-	min_players = 20
+#/datum/next_map/line
```
Comment (Kurfursten) [inline code/datums/next_map.dm:83]: This isn't how you comment in DM, use // for single line comments or /* (start) */ (stop) for multiline

Comment (west3436) [reply code/datums/next_map.dm:83]: Noted for future use, thanks

**Pull request closed without merging**